packages/tocco-theme/src/ToccoTheme/vendor-extension/toastr/third-party-integration.scss

```diff
@@ -35,6 +35,7 @@
   .bottom-center {
     max-height: 100%; // usability: ensure accessibility of all notification boxes
     overflow-y: auto; // todo: enhance usability - it is not obvious that content is scrollable
+    padding-bottom: 25px;  // reset: react-redux-toastr (index.scss)
```

Comment:
This fix causes other problems and must not be commited. Content beneath .top-left, .top-right, .top-center, .bottom-left, .bottom-right, .bottom-center is not accessible for actions.

Reply:
fixed by commit 0c8a27674402623d3e0e9d4db72e31e31b8835cb